packages/sdk-py/src/agent_relay/communicate/transport.py

```
@@ -392,19 +336,20 @@ async def _dispatch_ws_payload(self, raw_payload: str) -> None:
 
         result = callback(message)
         if isawaitable(result):
-            await result
+            # WsClient invokes handlers synchronously; schedule the coroutine.
+            asyncio.ensure_future(result)
```

Ruff (0.15.18)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)